lib/cbus_elixir_web/templates/page/index.html.eex

```diff
@@ -48,7 +48,7 @@
         <div class="sponsors">
             <h2 class="title myriad">Sponsors</h2>
             <div class="sponsors">
-                <a href="https://matchedpattern.com/" target="_blank">
+                <a href="http://www.matchedpattern.com/" target="_blank">
```

Comment:
Dropping SSL? This will still work, but just has an extra re-direct

Reply:
@Kartstig the ssl link does not work. It spins and spins and spins until it times out. We have had t update our link everywhere because of it. It was discussed in the various channels (most recently in Jon's healthcare app channel).

Reply:
I thought it works as long as www is provided (just tested in my browser and it works)